inc/Engine/CDN/RocketCDN/RESTSubscriber.php

```diff
@@ -74,16 +74,17 @@ public function register_enable_route() {
 					],
 					'url'   => [
 						'required'          => true,
-						'validate_callback' => function( $param ) {
+						'validate_callback' => function ( $param ) {
 							$url = esc_url_raw( $param );
 
 							return ! empty( $url );
 						},
-						'sanitize_callback' => function( $param ) {
+						'sanitize_callback' => function ( $param ) {
 							return esc_url_raw( $param );
 						},
 					],
 				],
+				'permission_callback' => __return_true(),
```

Comment:
Looking at the [documentation for permission_callback](https://developer.wordpress.org/rest-api/extending-the-rest-api/adding-custom-endpoints/#permissions-callback), it should be registered like so:

```suggestion
				'permission_callback' => '__return_true',
```